.travis.yml

```diff
@@ -1,9 +1,20 @@
 language: python
 python:
 - 2.7
+
 install:
 - pip install -r requirements-dev.txt
 - pip install .
+- pip install coveralls
```

Comment:
I guess so but we have it like that on the rest of the drivers so I wanted to be consistent. But I can fix it. No biggie.